packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -0,0 +1,178 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+
+import '../../all-components';
+import type { WaferMapDie, WaferMapColorsScale } from '../types';
+import {
+    WaferMapQuadrant,
+    WaferMapOrientation,
+    WaferMapColorsScaleMode
+} from '../types';
+import {
+    HIGHLIGHTEDVALUESETS,
+    WAFERMAPDIESETS,
+    WAFERMAPCOLORSCALESETS
+} from './sets';
+
+interface WafermapArgs {
```

Comment:
I think you can used "WaferMap" with "M" instead of "m" in all the places :) We consider "wafer map" 2 different words.

Reply:
Also in the "title" line 31 and "component" line 35 and "description" line 71 and other places if there are :)

Reply:
Applied